test/gtest/CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
-include(gtest) #include gtest.cmake
-enable_testing()
+find_package(GTest REQUIRED)
```

Comment:
This should probably do `find_package(GTest CONFIG REQUIRED)` to use the cmake config directly.(especially since cmake < 3.20 uses different names for the targets).